src/libsystemd/sd-bus/bus-socket.c

```diff
@@ -219,7 +217,7 @@ static int bus_socket_auth_verify_client(sd_bus *b) {
 
         b->auth = b->anonymous_auth ? BUS_AUTH_ANONYMOUS : BUS_AUTH_EXTERNAL;
 
-        for (i = 0; i < 32; i += 2) {
+        for (unsigned i = 0; i < 32; i += 2) {
```

Comment:
i'd probably have changed the type to size_t to, after all it's used for iterating through a string, i.e. memory. but i guess unsigned works fine too